services/google_search_service/request_test.go

```diff
@@ -5,48 +5,69 @@ import (
 	"strings"
 	"testing"
 
+	errorHelper "github.com/gutakk/go-google-scraper/helpers/error_handler"
+
 	"github.com/dnaeon/go-vcr/recorder"
+	log "github.com/sirupsen/logrus"
 	"gopkg.in/go-playground/assert.v1"
 )
 
 func TestRequestWithValidKeyword(t *testing.T) {
-	r, _ := recorder.New("tests/fixture/vcr/valid_keyword")
+	r, err := recorder.New("tests/fixture/vcr/valid_keyword")
+	if err != nil {
+		log.Error(errorHelper.RecordInitializeFailure, err)
+	}
+
+	resp, requestErr := Request("AWS", r)
 
-	resp, _ := Request("AWS", r)
+	err = r.Stop()
+	if err != nil {
+		log.Error(errorHelper.RecordStopFailure, err)
+	}
 
 	p, err := ioutil.ReadAll(resp.Body)
 	isGoogleSearchPage := err == nil && strings.Index(string(p), "<title>AWS") > 0
 
-	assert.Equal(t, nil, err)
+	assert.Equal(t, nil, requestErr)
 	assert.Equal(t, true, isGoogleSearchPage)
```

Comment:
@gutakk Do you think it would be easier to read if we handle the error from `ioutil.ReadAll` separately before assertion?
```Go
p, err := ioutil.ReadAll(resp.Body)
if err != nil {
  log.Error(...)
}

isGoogleSearchPage := strings.Index(string(p), "<title>AWS") > 0
assert.Equal(t, true, isGoogleSearchPage)
```

Reply:
Good catch. I will do it in https://github.com/gutakk/go-google-scraper/issues/79